packages/client-sdk-web/src/config/vector-configuration.ts

```diff
@@ -0,0 +1,48 @@
+import {Configuration} from './configuration';
+import {MomentoLoggerFactory} from '@gomomento/sdk-core';
+import {TransportStrategy} from './transport';
+
+export interface VectorConfigurationProps {
+  /**
+   * Configures logging verbosity and format
+   */
+  loggerFactory: MomentoLoggerFactory;
+
+  /**
+   * Configures low-level options for network interactions with the Momento service
+   */
+  transportStrategy: TransportStrategy;
+}
+
+export class VectorConfiguration implements Configuration {
```

Comment:
will we need these in nodejs SDK too? or does your PR not wire up node.js SDK yet?

Reply:
I'm not doing anything on the node side in this PR. This is specifically for Web SDK and console.

Reply:
do you have a sense of how much extra work it would be? I presumed it would be pretty small, and that we would knock them both out together (in fact I'd kind of prefer that we establish a pattern of always adding stuff to both at the same time and not letting them drift from one another).

not a blocker for this PR but if it seems easy and you have bandwidth maybe you could file a follow-up PR before you split.